test/nuttx-config.js

```diff
@@ -0,0 +1,122 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
```

Comment:
okay, renaming is a good idea! but the problem still remains like below, I wrote this driver to run on the device, and device cannot run throughout whole tests, and I don't know what's the best idea to workaround. 
The reason I divided tests to 8 groups, isn't becauses there's a special reason, but just to be small enough to run on the device, and switching the group help developers run whole tests. Maybe we need discussions what we can do the most for this.